### PR TITLE
Dockerised Python's backend

### PR DESCRIPTION
**Pull request by expobrain**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,3 +37,6 @@ public/main.css.map
 
 # Mac
 .DS_Store
+
+# Visual Studio Code
+.vscode
```
Comment (sztanko) [inline .gitignore]: Add .pyc as well

File: src/python/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+FROM anapsix/alpine-java:8_server-jre
+
+RUN apk update && apk upgrade
+RUN apk add python3
+RUN apk add g++ python3-dev
+RUN apk add curl tar perl
+
+RUN cd /usr/local && curl -l https://www.exasol.com/support/secure/attachment/55207/EXASOL_JDBC-6.0.4.tar.gz  | tar zxf -
```
Comment (sztanko) [inline src/python/Dockerfile]: Wouldn't it make sense to have version (6.0.4) as a variable?

File: src/python/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Flask==0.12.2
+Flask-Compress==1.4.0
+ujson==1.35
```
Comment (sztanko) [inline src/python/requirements.txt]: Is csv library a core one or should be added here as well?

File: .gitignore (hunk 1 above)
Comment (expobrain) [reply .gitignore]: `pyc` files are excluded as well in the same `.gitignore` file

File: src/python/Dockerfile (hunk 2 above)
Comment (expobrain) [reply src/python/Dockerfile]: yes, but the URL not only include the version number but also an attachment ID so I should have that in a variable as well, at this point IMHO is easier to just leave the entire URL inlined

File: src/python/requirements.txt (hunk 3 above)
Comment (expobrain) [reply src/python/requirements.txt]: `csv` package is part of the standard library https://docs.python.org/3/library/csv.html that's why is not included

**Pull request merged**